### PR TITLE
Make snapshots work for elasticsearch tests

### PR DESCRIPTION
**Pull request by diarmuidcc**
[#snapshots]

added new directory for snapshots and configuration variable needed to make snapshots work

### CODE REVIEW

**Comment by diarmuidcc** [comment]
@pemacc could you please review this?

**Comment by diarmuidcc** [comment]
@pemacc we cant seem to assign anyone...

**Comment by pemacc** [comment]
look good

File: 2.3/2.3.2/Dockerfile [hunk 1]
```diff
@@ -8,4 +8,7 @@ RUN plugin install marvel-agent
 RUN mkdir -p /artefacts/es
 RUN chmod 777 /artefacts
 
+RUN mkdir -p /usr/share/elasticsearch/snapshots
+RUN chmod 777 /usr/share/elasticsearch/snapshots
 
+COPY config /usr/share/elasticsearch/config
```
Comment (pemacc) [inline 2.3/2.3.2/Dockerfile:12]: why this line?

File: 2.3/2.3.2/Dockerfile [hunk 2]
```diff
@@ -8,4 +8,7 @@ RUN plugin install marvel-agent
 RUN mkdir -p /artefacts/es
 RUN chmod 777 /artefacts
 
+RUN mkdir -p /usr/share/elasticsearch/snapshots
+RUN chmod 777 /usr/share/elasticsearch/snapshots
```
Comment (pemacc) [inline 2.3/2.3.2/Dockerfile]: World writable?

**Pull request merged**